Neos.Utility.ObjectHandling/Classes/ObjectAccess.php

```diff
@@ -119,6 +120,11 @@ protected static function getPropertyInternal($subject, string $propertyName, bo
                 $propertyReflection->setAccessible(true);
                 return $propertyReflection->getValue($subject);
             }
+            if ($subject instanceof ProxyInterface && property_exists(get_parent_class($className), $propertyName)) {
```

Comment:
Would it maybe be more straight-forward if the check for ProxyInterface is done and only overwrites `$className` (inside the forceDirectAccess condition)? The access code is identical otherwise.